### PR TITLE
Fix player names not being reinitialized on change.

### PR DESCRIPTION
**Pull request by YelehaUwU**

### CODE REVIEW

File: [web]/performancebrowser/targetManager.lua [hunk 1]
```diff
@@ -26,6 +26,13 @@ addEventHandler('onNotifyTargetEnabled', resourceRoot,
 	end
 )
 
+addEventHandler('onPlayerChangeNick', root,
+	function(old, new, user)
+		if targetList[getPlayerKey(source)] then
```
Comment (Nico8340) [inline [web]/performancebrowser/targetManager.lua:31]: Please use the source variable here, as this event can only have a player as its source

File: [web]/performancebrowser/targetManager.lua [hunk 2]
```diff
@@ -26,6 +26,13 @@ addEventHandler('onNotifyTargetEnabled', resourceRoot,
 	end
 )
 
+addEventHandler('onPlayerChangeNick', root,
+	function(old, new, user)
+		if targetList[getPlayerKey(source)] then
+			targetList[getPlayerKey(source)].name = "client: "..new
```
Comment (Nico8340) [inline [web]/performancebrowser/targetManager.lua:32]: Please use the source variable here as well

**Review by YelehaUwU — COMMENTED** [review]
Fixed

**Review by Nico8340 — COMMENTED** [review]
LGTM

**Pull request merged**